
common:
    lid_topic:  "/livox/lidar"  # _192_168_3_111
    imu_topic:  "/livox/imu"
    filter_size_surf: 0.25
    filter_size_map: 0.25

kf:
    maximum_iter: 4
    epsi: 0.001
    NUM_MATCH_POINTS: 5
    plane_thr: 0.1 # 距离阈值

preprocess:
    lidar_type: 4             
    scan_line: 4
    blind: 2.0
    max_range: 40.0
    point_filter_num: 1

dyna:
    en: true
    ivox_resolution: 0.25
    use_chi: true
    chi_square_critical: 6  # 3.84 - 95%   2.71 - 90%   1.64 - 80%    1.07 - 70%    0.708 - 60%  0.455 - 50%

loc:
    en: true
    loadmap_dir: /home/qianliyue/D:/RepoMy/save/paper/mFFR/
    init_guess: [0.01768, -0.002955, -0.010476, 1.889178, 1.339936,0.029177] 

    
mapping:
    acc_cov: 0.1
    gyr_cov: 0.1
    b_acc_cov: 0.0001
    b_gyr_cov: 0.0001
    extrinsic_T: [ -0.011, -0.02329, 0.04412 ]
    extrinsic_R: [  1., 0., 0.,
                    0., 1., 0.,
                    0., 0., 1.]

publish:
    path_en:  true
    scan_publish_en:  true       
    dense_publish_en: false    
    runtime_pos_log: true


pcd_save:
    pcd_save_en: 0
    savemap_dir: /home/qianliyue/D:/RepoMy/save/paper/ltest/

